modules/oe_corporate_countries_address/tests/src/Kernel/AvailableCountriesSubscriberTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\Tests\oe_corporate_countries_address\Kernel;
+
+use Drupal\entity_test\Entity\EntityTest;
+use Drupal\field\Entity\FieldConfig;
+use Drupal\field\Entity\FieldStorageConfig;
+use Drupal\Tests\oe_corporate_countries\Kernel\CorporateCountriesRdfKernelTestBase;
+
+/**
+ * Tests the AvailableCountriesSubscriber class.
+ */
+class AvailableCountriesSubscriberTest extends CorporateCountriesRdfKernelTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'user',
+    'entity_test',
+    'address',
+    'oe_corporate_countries_address',
+  ];
+
+  /**
+   * Tests the deprecated countries are removed from the countries array.
+   */
+  public function testGetAvailableCountriesArray(): void {
+
+    $entityTypeName = 'entity_test';
+
+    FieldStorageConfig::create([
+      'field_name' => 'field_country',
+      'entity_type' => $entityTypeName,
+      'type' => 'address_country',
+    ])->save();
+
+    FieldConfig::create([
+      'label' => 'My field',
+      'field_name' => 'field_country',
```

Comment:
Field configs need a field storage (that you created above), a bundle and a label. They do not need a field name and entity type (these are on the storage).

Reply:
Okey, but if i remove these (field name and entity type) I would get these errors:

for removing field:
```
Drupal\Core\Field\FieldException: Attempt to create a field without a field_name.
```
for removing entity type:
```
Drupal\Core\Field\FieldException: Attempt to create a field 'field_country' without an entity_type.
```

So for now i would keep this code intact.

Reply:
This has not been removed.